docs/CARS.md

```diff
@@ -7,8 +7,8 @@
 | Acura     | ILX 2016-19                   | AcuraWatch Plus   | openpilot        | 25mph<sup>1</sup>  | 25mph             |
 | Acura     | RDX 2016-18                   | AcuraWatch Plus   | openpilot        | 25mph<sup>1</sup>  | 12mph             |
 | Acura     | RDX 2019-21                   | All               | Stock            | 0mph               | 3mph              |
-| Honda     | Accord 2018-20                | All               | Stock            | 0mph               | 3mph              |
-| Honda     | Accord Hybrid 2018-20         | All               | Stock            | 0mph               | 3mph              |
+| Honda     | Accord 2018-21                | All               | Stock            | 0mph               | 3mph              |
+| Honda     | Accord Hybrid 2018-21         | All               | Stock            | 0mph               | 3mph              |
```

Comment:
@csouers did we confirm the new ones stop steering under 3mph?

Reply:
Hmm. No, but I have access to a few users' devices. I'll see what I can find.

Reply:
So, yes! It's basically 0 mph. Steer angle stops changing at around 0.3 m/s (0.67 mph).

![Screenshot from 2021-11-24 13-17-14](https://user-images.githubusercontent.com/16481750/143293314-6b48062f-28a2-4771-a2a0-7f55bcbcb32b.png)
`./juggle.py 'f29e2b57a55e7ad5|2021-11-21--17-39-20' 117 4 --can`